tests/ttnn/unit_tests/operations/test_reduction_mean.py

```diff
@@ -26,3 +26,24 @@ def test_mean(device, batch_size, h, w, dim):
     output_tensor = ttnn.mean(input_tensor, dim=dim)
     output_tensor = ttnn.to_torch(output_tensor)
     assert_with_pcc(torch_output_tensor, output_tensor)
+
+
+@pytest.mark.parametrize("batch_size", [1, 16, None])
```

Comment:
COuld you add a test case which was described in the bug report? (64,4,1,32)

Reply:
Added, and verified that it's called:
```
tests/ttnn/unit_tests/operations/test_reduction_mean.py::test_mean_without_dim[w=32-h=1-batch_size=(64, 4)]
...
                     Op | DEBUG    | 	0: Tensor(storage=DeviceStorage(memory_config=MemoryConfig(memory_layout=TensorMemoryLayout::INTERLEAVED,buffer_type=BufferType::DRAM,shard_spec=std::nullopt)),shape=ttnn.Shape([64, 4, 1[32], 32]),dtype=DataType::BFLOAT16,layout=Layout::TILE,tile=Tile(tile_shape={32, 32},face_shape={16, 16},num_faces=4))
```